packages/active-campaign-client/src/__tests__/handlers/manageListSubscription.test.ts

```diff
@@ -0,0 +1,26 @@
+import { manageListSubscription } from '../../handlers/manageListSubscription';
+
+// remove .skip to run the test, be aware it does real API calls
+describe('manage list subscription', () => {
```

Comment:
What [skip](https://www.youtube.com/watch?v=lOjdy0K3X7s)? 
```suggestion
// remove .skip to run the test, be aware it does real API calls
describe.skip('manage list subscription', () => {
```